ivy/functional/extensions/__init__.py

```diff
@@ -19,8 +19,8 @@
 from .gradients import *
 from . import layers
 from .layers import *
-from . import linear_algebra
```

Comment:
The cause of the existence  `ivy.linear_algebra` issue was this line. Simply changing this line to `from . import linear_algebra as linalg` will resolve the issue. No need to rename all of them.

Reply:
I just tried to only change that line separately on a new branch from master, but the `ivy.linear_algebra` still persists. 

Adding that update will still show the following message:
```
>>> ivy.linear_algebra
<module 'ivy.functional.ivy.linear_algebra' from '/workspaces/ivy/ivy/functional/ivy/linear_algebra.py'>
```
However, if you change the `functional/ivy/linear_algebra.py` file, the link will now point to that module:
```
>>> ivy.linear_algebra
<module 'ivy.functional.extensions.linear_algebra' from '/workspaces/ivy/ivy/functional/extensions/linear_algebra.py'>
```

So it appears that we still need to make changes to both of the files.

The reason for that is that in the `ivy/functional/__init__.py` we are calling 
```
from .extensions import *
```
which obviously simply loads modules in that folder as they are named

However, in the `ivy/__init__.py` there is no such import, but even if you change the import format on line 533 to 
```
from ivy.functional.ivy import (
    ...
    linear_algebra as linalg,
    ...
)
```

It is still somehow getting the `linear_algebra` module to be exposed and the only way to resolve that was actually renaming the file.

So I still think we need to change both instances here, but please correct me if I am wrong. I tried different ways, but only that one worked for me, and probably I am missing something.